File: Blatt1/Blatt01-Beispielcode/Blatt01-Beispielcode/scala/CountDown.scala
@main def countDown = {       // Syntax kept close to JAVA
  for (i <- 10 to 1 by -1) {           // (for simplicity)
    println(i);
    if (i == 8) {
        println("ignition sequence start");
    }
  }
  println("liftoff!");
}

File: Blatt1/Blatt01-Beispielcode/Blatt01-Beispielcode/java/CountDown.java
public class CountDown {   // Klassenname (Modul) entspricht Dateiname
    public static void main(String[] args) {  // Standard Hauptprogram
        int i=10;                                  // declare variable
        while (i>0) {                   // while loop (for simplicity)
            System.out.println(i);                     // print number
            if (i == 8) {
                System.out.println("ignition sequence start");
            }
            i = i - 1;          // core code almost identical to C/C++
        }
        System.out.println("liftoff!");
    }
}